P3804.md

```diff
@@ -451,7 +451,7 @@ optional<P> try_query(Query q) @[`const`]{.add}@ noexcept;
 
 [5]{.pnum} *Mandates*: `P` is a cv-unqualified non-array object type.
 
-[6]{.pnum} *Returns*: Let `env` be the environment of the receiver represented by `*this` [ and `template <class T, class R> R` _`implementation-defined-transform`_`(T&&)` an implementation-defined transformation]{.add}.
+[6]{.pnum} *Returns*: Let `env` be the environment of the receiver represented by `*this`.
```

Comment:
I thought that Tomasz composed wording, sent via email, had this part about _implementation-defined-transform_ but I might be mistaken. I trust you double checked and made what is a right thing to do.

I expect the changes in this section in the future, anyway, so keep it as is (as it is in the isocpp.org portal).

looked at the changes and I know what we forgot. Adding `p` (former `parallelizable`) for implementation defined policies. I will try to quickly do that